gneiss/sort.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+import pandas as pd
+from functools import partial
+from gneiss.util import match
+
+
+def mean_niche_estimator(abundances, gradient):
+    """ Estimates the mean niche of an organism.
+
+    Calculates the mean niche or an organism along a gradient.
+    This is done by calculating the expected value of an organism
+    across the gradient.
```

Comment:
At first thought, I thought this was obvious.

But perhaps its a little confusing in its current state - definitely worth enabling sphinx docs before the next release.  Raising an issue.

For now, the MathJax notation will be added, but not rendered.